services/graph/pkg/config/reva.go

```diff
@@ -2,5 +2,6 @@ package config
 
 // TokenManager is the config for using the reva token manager
 type TokenManager struct {
-	JWTSecret string `yaml:"jwt_secret" env:"OCIS_JWT_SECRET;GRAPH_JWT_SECRET" desc:"The secret to mint and validate jwt tokens." introductionVersion:"pre5.0"`
+	JWTSecret              string `yaml:"jwt_secret" env:"OCIS_JWT_SECRET;GRAPH_JWT_SECRET" desc:"The secret to mint and validate jwt tokens." introductionVersion:"pre5.0"`
+	ShowUserEmailInResults bool   `yaml:"mask_user_email" env:"OCIS_SHOW_USER_EMAIL_IN_RESULTS" desc:"Mask user email addresses in responses." introductionVersion:"5.1"`
```

Comment:
Why did you put the setting in the `TokenManager` struct. I don't think it should go there.

Wouldn't it better fit into `config.API`? See config.go

The same it true for the `ocs` service btw.

Reply:
Oh, if it is still for debate if we need this switch or not, I am team Ralf :smiley:

Reply:
I already adapted the code